MSAL/src/MSALPublicClientApplication.m

```diff
@@ -365,8 +367,23 @@ + (BOOL)cancelCurrentWebAuthSession
     return NO;
 }
 
-#pragma mark -
-#pragma mark acquireToken
+#pragma mark - Acquire Token
+
+- (void)acquireTokenWithParameters:(MSALInteractiveTokenParameters *)parameters
+                   completionBlock:(MSALCompletionBlock)completionBlock
+{
+    [self acquireTokenForScopes:parameters.scopes
```

Comment:
nit: would it make sense to flip it? Have an internal method that always accepts parameters and all overloads can create a parameter object and pass it to that method?

Reply:
We will need to change it in many places (~13 overloads), but I don't see much benefit in doing this way. Let's discuss it and address in separate pr.